src/redux/hub/hub.reducer.ts

```diff
@@ -0,0 +1,31 @@
+import {HubState, HubActionTypes} from "./hub.types";
+import {
+  FETCH_HUB_IMAGES, FETCH_HUB_IMAGES_SUCCESS, FETCH_HUB_IMAGES_FAILURE, initialHubState
+} from "./hub.constants";
+
+export default function globalReducer(
+  state: HubState = initialHubState,
+  action: HubActionTypes
+): HubState {
+  switch (action.type) {
+    case FETCH_HUB_IMAGES:
+      return {
+        ...state,
+        loading: true
+      }
+    case FETCH_HUB_IMAGES_SUCCESS:
```

Comment:
In my opinion the action should describe what is happing. FETCH_HUB_IMAGES_SUCCESS is describing why it is happening. What is happening is SET_HUB_IMAGES, right?